fe-traveltrove/tests/cities.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { test, expect } from '@playwright/test';
+
+test('get all cities', async ({ page }) => {
+    await page.goto('http://localhost:3000/dashboard');
+    await page.getByRole('tab', { name: 'Cities' }).click();
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('New York');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Paris');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Rome');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Tokyo');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Toronto');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Berlin');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Sydney');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Mumbai');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Beijing');
+    await expect(page.getByLabel('Cities').locator('tbody')).toContainText('Rio de Janeiro');
+    await page.close()
+});
+
+test('view city details page', async ({ page }) => {
+    await page.goto('http://localhost:3000/dashboard');
+    await page.getByRole('tab', { name: 'Cities' }).click();
+    await page.getByRole('cell', { name: 'New York' }).click();
+    await page.close()
+});
```

Comment:
I would add the fact that you should be clicking in the 'cities' tab in the dashboard